c2rust-transpile/src/c_ast/mod.rs

```diff
@@ -130,6 +130,25 @@ impl TypedAstContext {
         }
     }
 
+    /// Predicate for (pointers to) structs that are binary compatible with C's `va_list`.
```

Comment:
Does this return true for both `struct __va_list_tag` and `struct __va_list_tag*`? If so, I'm not sure that's correct.